src/MSBuild/OutOfProcTaskHostNode.cs

```diff
@@ -1161,5 +1161,20 @@ private void LogErrorFromResource(string messageResource)
 
             LogErrorEvent(error);
         }
+
+        public int RequestCores(int requestedCores)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void ReleaseCores(int coresToRelease)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
I'm not sure what to do here in the OOP taskhost node.

Options:

1. As is, just throw (pros: done! cons: tasks don't necessarily know or care that they're in such a node)
2. Just return 1 (pros: easy! cons: doesn't do what it says on the tin)
3. Make sure the service is available here too and works (pros: ... works, cons: not sure how hard for not-huge benefit)

Reply:
For (3), why can't you just copy the code from TaskHost?